src/main/java/com/todolist/enterprise/dto/List.java

```diff
@@ -0,0 +1,8 @@
+package com.todolist.enterprise.dto;
+import lombok.Data;
+
+public @Data class List {
+    private int ListId;
```

Comment:
```suggestion
    private int listId;
```
Change variable to camelcase to match the rest of the variables as well as naming conventions.